changelogs/unreleased/3565-compatibility-iso4-modulev2.yml

```diff
@@ -0,0 +1,8 @@
+change-type: patch
+issue-nr: 3536
+description: Raise a warning when trying to use repo of type package.
+destination-branches: [iso4]
+sections:
+    bugfix: "{{description}}"
+    upgrade-note: "Creating new modules using the template should be done by first checking out on the v1 branch"
```

Comment:
Maybe we should mention the exact command here?

Reply:
I think we should also update the README of the inmanta-module-template repository with information that describes the procedure to create a V1 or a V2 module.

Reply:
@arnaudsjs 

currently the same procedure is set for v1 as v2 modules:

```
# Usage
cookiecutter https://github.com/inmanta/inmanta-module-template.git

This command will prompt for the template parameters.
```
do you propose to change it for V1 to:

`cookiecutter --checkout v1 https://github.com/inmanta/inmanta-module-template.git`?

Reply:
I think the README should give  clear overview regarding the features we support (which module versions it supports) and how it should be used (which command you should invoke). So we should mention the command for both V1 and V2 modules.

Reply:
I created a PR: https://github.com/inmanta/inmanta-module-template/pull/24

Reply:
I'll add to this that the upgrade note should mention `--checkout v1`:
"Creating new modules using the cookiecutter template should now requires the `--checkout v1` option."

For some additional context, currently this is not required yet, but we'll change this once we've released iso4. Then `--checkout v2` will no longer be required for v2 modules.

Reply:
@FloLey I think you forgot to apply the change request on the `update-note`. Is that correct?